pkg/config/config.go

```diff
@@ -68,7 +68,7 @@ var (
 	EnabledGPU                   = getBoolConfig("ENABLE_GPU", false)
 	EnableProcessMetrics         = getBoolConfig("ENABLE_PROCESS_METRICS", false)
 	ExposeHardwareCounterMetrics = getBoolConfig("EXPOSE_HW_COUNTER_METRICS", true)
-	ExposeCgroupMetrics          = getBoolConfig("EXPOSE_CGROUP_METRICS", true)
+	ExposeCgroupMetrics          = getBoolConfig("EXPOSE_CGROUP_METRICS", false)
```

Comment:
ebpf and cgroup in a same PR?

Reply:
we'll discuss it in the planning meeting